airflow/settings.py

```diff
@@ -722,6 +722,22 @@ def import_local_settings():
         log.info("Loaded airflow_local_settings from %s .", airflow_local_settings.__file__)
 
 
+def mask_conf_values():
+    from airflow.utils.log.secrets_masker import mask_secret
+
+    for section, key in conf.sensitive_config_values:
+        try:
+            value = conf.get(section, key)
+        except AirflowConfigException:
+            log.warning(
+                "Could not retrieve value from section %s, for key %s. Skipping redaction of this conf.",
+                section,
+                key,
+            )
+            continue
```

Comment:
How many messages is this going to create every time Airflow starts up? This would also appear inside the task instance logs, so I think this should probably be reduced to a debug level log so that it doesn't "spam" things.

Reply:
Makes sense, this will log a couple of times.
I will make this change and move it to DEBUG so that the log flooding is much lesser.